server/aws-lsp-partiql/README.md

```diff
@@ -10,6 +10,8 @@ _Updating the binary requires the Docker CLI to be in your PATH_
 
 To update the binary used by the server, run `npm run update-parser-binary`. This will, inside a Docker container, pull the PartiQL playground repository from GitHub and run the `build` command to create a new binary. It is then encoded into base64 in a Typescript file and copied to the `src` folder. Running `npm run build` will then transpile the binary file to Javascript and place it in the `lib` folder.
 
+To update the binary used in package `web-tree-sitter`, run `npm run update-treesitter-wasm`. This update is forced to make everytime upgrading the version of `web-tree-sitter`. This will compile the `tree-sitter.wasm` file inside the package into a base64 string in a TypeScript file to the `src/tree-sitter-parser` folder as `tree-sitter-inline.ts`. 
```

Comment:
Have you considered excluding generated files from `prettier`, `eslint`?

Reply:
Yes, that's a great suggestion to keep our repository cleaner. I've added the generated files to the `.prettierignore` and `.eslintignore` configurations to prevent them from being unnecessarily reformatted or linted. Thank you for pointing this out!